src/sentry/replays/post_process.py

```diff
@@ -33,7 +41,7 @@ class BrowserResponseType(TypedDict, total=False):
 
 
 class UserResponseType(TypedDict, total=False):
-    id: str | None
+    id: str | None  # string uint64
```

Comment:
This id field is not a uint64.  Its a string.  It comes from the customer's application.